### PR TITLE
sana-ajani/issue19

### LINKED ISSUE #19: @sana-ajani Add a nifty description
Can you add a description here so that people know what to do? I don't know what to do.

### PR DESCRIPTION
**Pull request by sana-ajani**
Fixes #19

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -9,10 +9,13 @@
                 <span class="has-text-primary">Group</span> links,
                 <span class="has-text-primary">Save</span> &
                 <span class="has-text-primary">Share</span> them with the
```
Comment (burkeholland) [inline src/views/Home.vue:11]: Can we add a break after the main title. Just a little more space would be nice.

Comment (burkeholland) [inline src/views/Home.vue:11]: Add a break

**Comment by sana-ajani** [comment]
> Add a break 

Done!

**Comment by burkeholland** [comment]
LGTM

**Pull request merged**